tee-worker/omni-executor/intent/executors/cross-chain/src/cross_chain_swap_tests.rs

```diff
@@ -317,9 +315,9 @@ async fn simple_cross_chain_swap_sol_to_bsc() {
 	let binance_api = Arc::new(binance_api_mock);
 	let bsc_client = Arc::new(bsc_client_mock);
 	let solana_client = Arc::new(solana_client_mock);
-	let accounting_contract_client: Arc<Box<dyn AccountingContractApi>> =
+	let accounting_contract_client: Arc<Box<dyn AccountingContractApi<Address, U256>>> =
```

Comment:
Can you call it `evm_ accounting_contract_client` to make it matching the solana variant?